### ISSUE TITLE
Docs: Update User Guide to use `pip install <extension>`

### ISSUE THREAD

**Issue by AshleyHenry15** [rstudio/rstudio#13479]
[Slack origin](https://positpbc.slack.com/archives/CUC92M4TT/p1682691867769249)

David Aja
  [3 months ago](https://positpbc.slack.com/archives/CUC92M4TT/p1682691867769249)
I think if managing JupyterLab Extensions [via the extensions manager](https://docs.posit.co/ide/server-pro/user/jupyter-lab/guide/extensions.html#extensions-manager) is not possible on Workbench, that should be the highest priority text in the extensions manager section, rather than a callout box?

Ashley Henry
(https://positpbc.slack.com/archives/CUC92M4TT/p1682693060826239?thread_ts=1682691867.769249&cid=CUC92M4TT)
I like having the callout box but it shouldn't be a warning - it should be an "important" and I think bumping it up to the top makes sense. Do you think that is sufficient 

David Aja
  [3 months ago](https://positpbc.slack.com/archives/CUC92M4TT/p1682693705076319?thread_ts=1682691867.769249&cid=CUC92M4TT)
I would probably put the directions for installing extensions via pip first, but raising the note to the top is also a good move.

Tom Mock
:spiral_calendar_pad:  [3 months ago](https://positpbc.slack.com/archives/CUC92M4TT/p1682693757249139?thread_ts=1682691867.769249&cid=CUC92M4TT)
I agree w/ David that we want to explicitly push people towards pip install <extension> , not even as a note, but rather as the PRIMARY method. When we upgrade to JupyterLab 4.0, there won’t be another option, as it will actually remove the ability to install via the extensions manager.

**Comment by AshleyHenry15** [comment]
Mentioning @jthomasmock

**Comment by jthomasmock** [comment]
Thanks Ashley! David has captured the issue well in that we REALLY want to emphasize the `pip install` method and de-emphasize the "Install" UI.

**Comment by ronblum** [comment]
Is this for the Workbench User Guide, since it's related to JupyterLab?

**Comment by jthomasmock** [comment]
Yes, this should be for the Workbench User Guide, specifically under JupyterLab section.

Once we switch to JupyterLab 4: https://github.com/rstudio/rstudio-pro/issues/4585 `pip install` will actually be the ONLY method to install extensions.

**Comment by luahan77m** [comment]
Hello, I want to help you update the guide. Can you assign it to me?
